doc/sphinx/inter_bonded.rst

```diff
@@ -426,7 +426,7 @@ temperature and friction coefficient.
 The bond is configured with::
 
     from espressomd.interactions import ThermalizedBond
-    thermalized_bond = ThermalizedBond(temp_com = <float>, gamma_com = <float>, temp_distancedrude = <float>, gamma_drude = <float>, k = <float>, r_cut = <float>)
+    thermalized_bond = ThermalizedBond(temp_com = <float>, gamma_com = <float>, temp_distance = <float>, gamma_distance = <float>, r_cut = <float>)
     system.bonded_inter.add(drude_bond)
```

Comment:
This should be system.bonded_inter.add(thermalized_bond)

Reply:
ahh yes..thx